examples/audit_log_export/README.md

```diff
@@ -0,0 +1,671 @@
+}
+new_auditlogexport = {
+  "auditlog_download_url" = tostring(null)
+  "cluster_id" = "41938b91-66ed-4c84-b70c-55c3f0ae4266"
+  "created_at" = "2024-03-13 06:48:43.596764031 +0000 UTC"
+  "end" = "2024-03-13T06:44:15+00:00"
+  "expiration" = tostring(null)
+  "id" = "b173a42e-3c2c-4245-af81-f8615d63b5ce"
+  "organization_id" = "637cea1d-fce5-40a5-9d48-8d0690e656ee"
+  "project_id" = "6aa13dbf-69cb-48e3-97af-d89f57ea7f90"
+  "start" = "2024-03-13T04:44:15+00:00"
+  "status" = tostring(null)
+}
+```
+
+### View the audit log export job
+
+Command: `terraform plan`
+
+Sample Output:
+```
+terraform plan
+╷
+│ Warning: Provider development overrides are in effect
+│
+│ The following provider development overrides are set in the CLI configuration:
+│  - couchbasecloud/couchbase-capella in /Users/hiteshwalia/GolandProjects/terraform-provider-couchbase-capella/bin
+│
+│ The behavior may therefore not match any released version of the provider and applying changes may cause the state to become incompatible with
+│ published releases.
+╵
+data.couchbase-capella_audit_log_export.existing_auditlogexport: Reading...
+couchbase-capella_audit_log_export.new_auditlogexport: Refreshing state... [id=b173a42e-3c2c-4245-af81-f8615d63b5ce]
+data.couchbase-capella_audit_log_export.existing_auditlogexport: Read complete after 0s
+
+Note: Objects have changed outside of Terraform
+
+Terraform detected the following changes made outside of Terraform since the last "terraform apply" which may have affected this plan:
+
+  # couchbase-capella_audit_log_export.new_auditlogexport has changed
+  ~ resource "couchbase-capella_audit_log_export" "new_auditlogexport" {
+      + auditlog_download_url = "https://cb-audit-logs-637cea1d-fce5-40a5-9d48-8d0690e656ee.s3.amazonaws.com/export/cluster-audit-logs-41938b91-66ed-4c84-b70c-55c3f0ae4266-from-2024-03-13-to-2024-03-13.tar.gz?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAT277BFQVIKDGOEO3%2F20240313%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240313T064846Z&X-Amz-Expires=259200&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEE8aCXVzLWVhc3QtMSJIMEYCIQDM2n%2BpV6cnfeTrSnvjlcl15s%2FoyMcYLVfljG4BsTYyFAIhAO93hNcpnJUKqtiKj%2BFxF7p5amM1IFDCvyki2UYdCgLMKqACCFgQAxoMMjY0MTM4NDY4Mzk0IgxvjWc3jbG5idbrLWQq%2FQGvHYupgMUmqlFHUvNH1OKIn9sZby26SKtlKiV57%2B8%2BiYZDsztbnB%2FgwZKcLYVWwq7zGvGCqeZEk1%2FgUitGJ3Vj0tiSjlTmwnQCYAwkaZqMMUH4m%2BkM66S8jfMtKIA1RAwTTJ1T2%2FoYVBDe%2FsbORrYiPyN4DfhnKRZySeTSy1cXaYedVzjiA%2BYugUwUB2GwDbaofWofE6Ho2%2FEP5dkfVWs0iHwxVFlYSaq7aJFCl%2BKj8hXhlmujNVeMW%2Bd6T6MHcwSMf2%2B429Pnhm%2BYc00eVOQxOEyvjyMA4V%2FuYDOv5coTkTg5y4m0EHVYve8tsro67UpdMzrumwbFoUOrw7WUMMyYxa8GOpwB8SvDoGuX3OERsbyY%2FIelecnQtQW4wvdGBUdZ2OcmS19dMQjmCuKBO4aKvGpWYpeuUouj9zgYcgUzC39xBMUwtrAgGRn0horrc7tGJD2CHbR1UOIbu7pJSw%2FZOvNTw%2Bg7w%2F8tfgMg9f7Iobnfsi2xYbREquAGShEWi64tC0kf5rCOFFacM%2BZm3tq3x8sbT%2FJ87djItmfJk%2FYf2y9W&X-Amz-SignedHeaders=host&X-Amz-Signature=7eb266db82ee7af58b42d8b6c2daa2ceb4f435b07c503fc839edff460b327d4e"
```

Comment:
This URL seems to contain a lot of AWS credentials and tokens. It's better to mask this information.